unitary/alpha/quantum_world_test.py

```diff
@@ -284,6 +284,50 @@ def test_undo(simulator, compile_to_qubits):
     assert all(result[0] == Light.GREEN for result in results)
 
 
+def test_copy():
```

Comment:
Can you also add the `compile_to_qubits` parameterization to ensure that copying does not break that somehow?

Reply:
Done.

Reply:
Still need to initialize the `QuantumWorld` with the param.

Reply:
Oh yeah, of course I do.  Thanks for all the reviews!

Reply:
No problemo!